src/ontology/mondo.Makefile

```diff
@@ -655,7 +655,8 @@ update_deprecated_mappings: $(ALL_SOURCES_DEPRECATED_PATTERNS)
 deprecated_annotation_merging:
 	sed -i 's/source="MONDO:equivalentObsolete",\ source="MONDO:obsoleteEquivalentObsolete"/source="MONDO:obsoleteEquivalentObsolete"/g' mondo-edit.obo || true
 	sed -i 's/source="MONDO:equivalentObsolete",\ source="MONDO:equivalentTo"/source="MONDO:equivalentObsolete"/g' mondo-edit.obo || true
-	#(.*)source="MONDO:equivalentObsolete"(.*)source="MONDO:obsoleteEquivalentObsolete"(.*) // 
+	sed -i '' 's/\(.*\)source="MONDO:equivalentObsolete"\(.*\)source="MONDO:obsoleteEquivalentObsolete"\(.*\)/\1source="MONDO:obsoleteEquivalentObsolete"\2\3/g' mondo-edit.obo || true
```

Comment:
This is the third `sed` in `deprecated_annotation_merging`. I haven't thought through all the ways in which the output of those first two might interact with what I've added. I'm assuming these changed commands play well together, but your review is appreciated.

Reply:
I've since replaced `sed -i ''` with `sed -i` for consistency. Note that on my Macbook, I needed `sed -i ''` to get it to work ([stackoverflow](https://stackoverflow.com/a/5694430/5258518)).